src/questions/blockchain_basics.yaml

```diff
@@ -1,35 +1,1100 @@
+  difficultyLevel: "Low"
+  choices:
+    - content: Kotlin
+      key: A
+      order: 0
+    - content: tcl/tk
+      key: B
+      order: 1
+    - content: Python
+      key: C
+      order: 2
+    - content: Solidity
+      key: D
+      order: 3
+- uuid: e599a418-f347-498a-bfed-ba546766a739
   type: MultipleChoice
-  # Content should be in Markdown format
-  content: What does SW DAO offer?
-  hint: This is shown if user needs some help
-  explanation: This is shown when the user answers the choice incorrectly
+  content: Which ideas were combined by Satoshi Nakamoto to introduce Bitcoin?
+  hint: Properties of Bitcoin
+  explanation: Bitcoin works on the principle of Proof of work and blockchain.
+  answerKeys:
+    - A
+    - B
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Proof of work
+      key: A
+      order: 0
+    - content: Blockchain
+      key: B
+      order: 1
+    - content: Mining pools
+      key: C
+      order: 2
+    - content: Proof of stake
+      key: D
+      order: 3
+- uuid: 29994d40-0576-4bc1-ba4c-59cd4934a715
+  type: SingleChoice
+  content: Which of the following statements are false regarding Bitcoin blockchain?
+  hint: Properties of Bitcoin and blockchain
+  explanation: Mistakes cannot be edited on Bitcoin because these documents are immutable.
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Mistakes can be edited and fixed
+      key: A
+      order: 0
+    - content: Censorship Resistant
+      key: B
+      order: 1
+    - content: Computational platform for smart contracts
+      key: C
+      order: 2
+    - content: Allows for portability of bitcoin
+      key: D
+      order: 3
+- uuid: 09516d24-7580-45e9-babd-58a9cf2bbc1a
+  type: MultipleChoice
+  content: Blockchain is a ...... system
+  hint: Properties of Blockchain
+  explanation: Blockchain system is decentralised and Distributed network.
+  answerKeys:
+    - B
+    - C
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Centralised
+      key: A
+      order: 0
+    - content: Decentralised
+      key: B
+      order: 1
+    - content: Distributed
+      key: C
+      order: 2
+    - content: None of the above
+      key: D
+      order: 3
+- uuid: 72878ee1-93f4-4b5f-a72f-92410db73b80
+  type: SingleChoice
+  content: Cryptographic hashing can also be termed as encryption.
+  hint: NoHint
+  explanation: Hashing is only one-way process that is private key can be converted to public key but vice versa not possible.
+  answerKeys:
+    - B
+  subTopics:
+    - "hashing"
+  difficultyLevel: "Low"
+  choices:
+    - content: True
+      key: A
+      order: 0
+    - content: False
+      key: B
+      order: 1
+- uuid: 8bc6e329-f119-430c-84a0-01aaf01433fe
+  type: MultipleChoice
+  content: What is most common problem with simple hashes?
```

Comment:
Add more context to `simple`. Not tech people don't know what are simple and non-simple hashed